src/test/java/pl/chojnacki/AppTest.java

```diff
@@ -1,20 +1,20 @@
-package pl.chojnacki;
-
-import static org.junit.Assert.assertTrue;
-
-import org.junit.Test;
-
-/**
- * Unit test for simple App.
- */
-public class AppTest 
-{
-    /**
-     * Rigorous Test :-)
-     */
-    @Test
-    public void shouldAnswerWithTrue()
-    {
-        assertTrue( true );
-    }
-}
+//package pl.chojnacki;
+//
+//import static org.junit.Assert.assertTrue;
+//
+//import org.junit.Test;
+//
+///**
```

Comment:
A to co tutaj robi? :P 1 zadanie - 1 pull request. Czemu w tym zadaniu znajduje się coś więcej niż sam .gitignore? ;)

Reply:
// komentarz - klasyczne zagranie :D Stworzyłem new branch nie przechodząc na master, stąd branch master->task2->task3, zgadza się? Stworzyć nowego branch-a task3 czy można jakoś to edytować jeszcze?